src/theme/components/hero.ts

```diff
@@ -105,8 +105,65 @@ const getSecondaryVariantStyles = (bgColor: string = "") => {
     },
   };
 };
+const getTextColor = (type, mode, foregroundColor, isDarkText) => {
+  const prefix = mode === "dark" ? "dark.ui." : "ui.";
+  const colorLight = foregroundColor
+    ? foregroundColor
+    : isDarkText
+    ? `${prefix}typography.${type}`
+    : `${prefix}typography.inverse.${type}`;
+  const colorDark = foregroundColor
+    ? foregroundColor
+    : isDarkText
+    ? `${prefix}typography.inverse.${type}`
+    : `${prefix}typography.${type}`;
+  const colorLink = foregroundColor
+    ? foregroundColor
+    : isDarkText
+    ? `ui.link.primary`
+    : `dark.ui.link.primary`;
+  const finalColor =
+    type === "link" ? colorLink : mode === "dark" ? colorDark : colorLight;
+  return finalColor;
+};
+const getLinkColor = (state, mode, foregroundColor, isDarkText) => {
+  let finalColor;
+  const isDarkMode = mode === "dark" ? true : false;
+  switch (state) {
+    default:
```

Comment:
Conventionally, the default case goes last in the list of cases.

Reply:
Hmm it might also help to make it:
```
case "default":
default:
  ....
```

That way it's explicit that the "default" case is also the default for the switch statemetn.

Reply:
Right.  Forgot about that.  It's been a while since I used a switch-case statement.